packages/jsii-pacmak/lib/targets/go/types/class.ts

```diff
@@ -140,9 +140,10 @@ export class ClassMethod implements GoTypeMember {
     const name = this.name;
 
     const instanceArg = this.parent.name.substring(0, 1).toLowerCase();
+    const returnTypeString = this.reference?.void ? '' : ` ${this.returnType}`;
```

Comment:
any reason not to incorporate this logic in the `returnType` getter? That was the point of https://github.com/aws/jsii/pull/2001

Reply:
The main reason was consistency. To only add spacing to the class returnType which affects how others call it, [for example the runtime call](https://github.com/aws/jsii/blob/master/packages/jsii-pacmak/lib/targets/go/runtime.ts#L34). I think `returnType` should stay generic enough to be useful outside of the class and not concerned with formatting.